lib/logstash/outputs/cloudwatch.rb

```diff
@@ -303,7 +303,7 @@ def count(aggregates, event)
         METRIC => (fmetric ? fmetric : event.sprintf(@metricname)),
         DIMENSIONS => dims,
         UNIT => unit,
-        TIMESTAMP => event.sprintf("%{+YYYY-MM-dd'T'HH:mm:00Z}")
+        TIMESTAMP => event.get('timestamp')
```

Comment:
`just a simple fix to use the event.get method to set the timestamp from the @timestamp`

Shouldn't this then be `event.get('@timestamp')` (or even `event.timestamp`)?